src/Leela.cpp

```diff
@@ -347,6 +347,8 @@ int main (int argc, char *argv[]) {
     auto komi = 7.5f;
     maingame->init_game(BOARD_SIZE, komi);
 
+    Network::init_rotation_table(*maingame);
```

Comment:
Why not let Network call this? Why does main have to know about it?

Reply:
@gcp (since this comment has been hidden by github)